assets/app/scripts/controllers/projects.js

```diff
@@ -19,4 +20,37 @@ angular.module('openshiftConsole')
         $scope.emptyMessage = "No projects to show.";
       });
     });
+
+    // Test if the user can submit project requests. Handle error notifications
+    // ourselves because 403 responses are expected.
+    DataService.get("projectRequests", null, $scope, { errorNotification: false})
+    .then(function() {
+      $scope.canCreate = true;
+    }, function(result) {
+      $scope.canCreate = false;
+
+      var data = result.data || {};
+
+      // 403 Forbidden indicates the user doesn't have authority.
+      // Any other failure status is an unexpected error.
+      if (result.status !== 403) {
+        var msg = 'Failed to determine create project permission';
+        if (result.status !== 0) {
+          msg += " (" + result.status + ")";
+        }
+        Logger.warn(msg);
+        return;
+      }
+
+      // Check if there are detailed messages. If there are, show them instead of our default message.
+      if (data.details) {
```

Comment:
@deads2k moving discussion about message here. there are three cases where we get 403:
1. no access to list projectRequests. the returned error message isn't suitable to display
2. access to list projectRequests, but no access to create projects. the default message isn't good to display today... we could probably come up with a better default.
3.  no access with a custom message set.

I could see coming up with a better message for 2, but the trick is distinguishing between those and 1. Putting the message we wanted the UI to surface in details was the quickest way I could think of to signal that. Open to other ideas.